cimsparql/queries.py

```diff
@@ -331,6 +331,7 @@ def load_query(
         where_list += sup.bid_market_code_query(mrid_subject)
 
     if nodes:
+        variables.append("(xsd:boolean(?connected) as ?status)")
```

Comment:
We probably have this other places too, but this assumes that `xsd` namespace exists. Thus, on startup we should probably ensure that `xsd` exists. (Note that this schema is also commonly called just `xs`(https://www.w3schools.com/xml/schema_intro.asp)  so we should check that it is indeed referred to as `xsd`. E.g.

```python
if "xsd" not in self.prefixes:
    raise ValueError("CimSparql require that the namespace is http://www.w3.org/2001/XMLSchema is available with prefix xsd")